src/main/java/view/GameView.java

```diff
@@ -71,7 +71,7 @@ private void updateGameWindow(GameState state) {
             int width = thumbIcon.getIconWidth();
             int height = thumbIcon.getIconHeight();
             
-            if (width > 1000 || height > 1000) {
+            if (width > 750 || height > 750) {
                 width *= 0.3;
                 height *= 0.3;

```

Comment:
Can we maybe adjust it to the same pixel size instead of 0.3x if some particular thumbs are too big